assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -77,6 +80,13 @@ const getContribAttrs = ({ contribType, contribAmount, intCmp }): ContribAttrs =
 
   const ctaLink = `${ctaLinks[contType]}?${params.toString()}`;
 
+  if (!addingMonthlyTest) {
+
+    const subheading = 'Support the Guardian’s editorial operations by making a one-off contribution today';
```

Comment:
Just curious: is 'editorial operations' the approved language? Seems a bit overly corporate to me.